apps/web/utils/fhir.ts

```diff
@@ -0,0 +1,26 @@
+import { FhirResourceType, FhirResourceTypes } from "@/constants/fhirResourceTypes";
+
+export async function isFhirCompliant(file: File): Promise<FhirResourceTypes> {
```

Comment:
I can break the code by uploading a huge mp4 file lol. we should put some guardrails even before checking if the file is complaint. wdyt? 

Example:
```suggestion
export async function isFhirCompliant(file: File): Promise<FhirResourceTypes> {
  if (!file.type.includes('application/json') && !file.name.endsWith('.json')) {
    alert("Please upload a JSON file containing FHIR data.");
    return FhirResourceType.NOT_SUPPORTED;
  }
```